src/libraries/System.Data.Common/src/System/Data/ForeignKeyConstraint.cs

```diff
@@ -761,7 +761,7 @@ public virtual AcceptRejectRule AcceptRejectRule
             DataColumn[] columns = new DataColumn[keys];
             DataColumn[] relatedColumns = new DataColumn[keys];
 
-            int iDest = 0;
+            int iDest;
```

Comment:
This can be declared in the loop where it's assigned.

Reply:
Done